packages/api-explorer/src/components/SideNav/SideNavMethods.tsx

```diff
@@ -70,6 +68,13 @@ const SideNavMethodsLayout: FC<MethodsProps> = ({
     if (_isOpen) history.push(`/${specKey}/methods/${tag}`)
   }
 
+  useEffect(() => {
+    const status = match
```

Comment:
setIsOpen defaults to defaultOpen but you are modifying the value if the value of defaultOpen changes. Is defaultOpen a good name?

Reply:
We can find a better name for another PR